tendrl/node_agent/objects/cluster_message/__init__.py

```diff
@@ -0,0 +1,25 @@
+from tendrl.commons import etcdobj
+from tendrl.commons.message import Message as message
+from tendrl.node_agent import objects
+
+
+class ClusterMessage(message, objects.NodeAgentBaseObject):
+    def __init__(self, **cluster_message):
+        super(ClusterMessage, self).__init__(**cluster_message)
+
+        self.value = 'clusters/%s/Messages/%s'
+        self._etcd_cls = _ClusterMessageEtcd
+
+
+class _ClusterMessageEtcd(etcdobj.EtcdObj):
+    """Cluster message object, lazily updated
+
+    """
+    __name__ = 'clusters/%s/Messages/%s'
+    _tendrl_cls = ClusterMessage
+
+    def render(self):
+        self.__name__ = self.__name__ % (
+            self.cluster_id, self.message_id
```

Comment:
@r0h4n you can see i have inherited both message and node_agent base class, so no need to give message variables again here, how it will work means cluster_message inherited from Message and Message inherited from node_agent.base, so it will work as we excepected